src/views/Reception.vue

```diff
@@ -77,30 +96,46 @@ export default {
       if (response.successful) {
         this.barcodes = response.deserialize.requests.map(r => r.barcode).join('\n')
       } else {
-        throw 'Failed to create tubes in Traction: ' + response.errors.message
+        throw Error(consts.MESSAGE_ERROR_CREATE_TUBES_FAILED + response.errors.message)
       }
     },
     async handleTractionTubes () {
```

Comment:
Again I am wondering if there is scope for a bit of refactoring? The method is quite lengthy and there seems to be a fair amount of logic checking for the material type.